modules/pairIdSystem.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * This module adds PAIR Id to the User ID module
+ * The {@link module:modules/userId} module is required
+ * @module modules/pairIdSystem
+ * @requires module:modules/userId
+ */
+
+import { submodule } from '../src/hook.js';
+import {getStorageManager} from '../src/storageManager.js'
+import { logError } from '../src/utils.js';
+
+const MODULE_NAME = 'pairId';
+const PAIR_ID_KEY = 'pairId';
+const DEFAULT_LIVERAMP_PAIR_ID_KEY = '_lr_pairId';
+
+export const storage = getStorageManager()
+
+function pairIdFromLocalStorage(key) {
+  return storage.localStorageIsEnabled ? storage.getDataFromLocalStorage(key) : null;
```

Comment:
`storage.localStorageIsEnabled` is a function, so this check will always evaluate to true.

This should be updated to `storage.localStorageIsEnabled()` which returns true or false.

Same with `storage.cookiesAreEnabled`